from django.contrib.auth import validators
from django.db import migrations, models


class Migration(migrations.Migration):
<<<<<<< HEAD

    dependencies = [
        ('auth', '0006_require_contenttypes_0002'),
=======
    dependencies = [
        ("auth", "0006_require_contenttypes_0002"),
>>>>>>> 32e7214c1350fb4a88064960159c92207483e581
    ]

    operations = [
        migrations.AlterField(
<<<<<<< HEAD
            model_name='user',
            name='username',
            field=models.CharField(
                error_messages={'unique': 'A user with that username already exists.'},
                help_text='Required. 30 characters or fewer. Letters, digits and @/./+/-/_ only.',
                max_length=30,
                unique=True,
                validators=[validators.UnicodeUsernameValidator()],
                verbose_name='username',
=======
            model_name="user",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text=(
                    "Required. 30 characters or fewer. Letters, digits and @/./+/-/_ "
                    "only."
                ),
                max_length=30,
                unique=True,
                validators=[validators.UnicodeUsernameValidator()],
                verbose_name="username",
>>>>>>> 32e7214c1350fb4a88064960159c92207483e581
            ),
        ),
    ]
